Update to variants (#816)
greenelab/covid19-review: content/09.evolution.md

By using tools from natural language processing (NLP), viral escape was modeled as an NLP problem [@doi:10.1126/science.abd7331] where a modification makes a sentence grammatically correct but semantically different.
Therefore, language models of viruses could predict mutations that change the presentation of the virus to the immune system but preserve its infectivity.

#### Variants of concern and variants under surveillance

Viral replication naturally leads to the occurrence of mutations that lead to genetic variants [@doi:10.1038/s41564-020-0690-4].
The emergence of new genetic variants of SARS-CoV-2 has attracted significant media attention since the detection of a variant of concern (VOC), a distinct phylogenetic cluster referred to as B.1.1.7/VOC 202012/01.

> While the B.1.1.7 lineage garnered attention in November 2020, two genomes of the lineage were detected as early as September 20th 2020 from routine genomic data in Kent, in the United Kingdom (UK) by the COVID-19 Genomics UK Consortium (COG-UK). 

Author: Probably, I wasn't sure

Since then, B.1.1.7 has spread across the UK and internationally where it has now been detected in at least 62 countries [@url:https://cov-lineages.org/global_report_B.1.1.7.html], despite several countries imposing travel restrictions on travellers from the UK. 
Of the twenty-three mutations than define B.1.1.7 from the original strain isolated in Wuhan (lineage A), fourteen are lineage-specific and three appear to be biologically consequential mutations associated with the spike protein, namely N501Y, P681H, and 69-70del [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf;  @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].

> The latter is a 6bp deletion leading to the loss of two amino acids with consequences for immune recognition, which may be responsible for increased transmissibility of the B.1.1.7 VOC in conjunction with N501Y due to changes in the RBD that mediates viral entry [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/947048/Technical_Briefing_VOC_SH_NJL2_SH2.pdf; @doi:10.1101/2020.12.14.422555]. 

Author: I agree with you @rando2 , nice way of putting it. The one thing on our side is the versatility of the mRNA vaccine design. With regards to above, I guess we can adapt part of this section from pathogenesis in order to explain the potential implications of mutation to the Spike. What do you think?

The FDA has highlighted that the performance of three diagnostic tests may be affected by the B.1.1.7 lineage by causing false negative tests [@url:https://www.fda.gov/medical-devices/letters-health-care-providers/genetic-variants-sars-cov-2-may-lead-false-negative-results-molecular-tests-detection-sars-cov-2].

 
While B.1.1.7 is currently the main VOC, other genetic variants not designated VOCs have been detected including B.1.351 (20H/501Y.V2) and P.1 ( 20J/501Y.V3), both of which emerged independently [@url:https://www.cdc.gov/coronavirus/2019-ncov/more/science-and-research/scientific-brief-emerging-variants.html].
B.1.351 was first detected in October 2020 in South Africa, which was later detected in the EU December 28th 2020 and has now spread to at least 26 countries [@url:https://www.gov.za/speeches/minister-zweli-mkhize-confirms-8-725-more-cases-coronavirus-covid-19-18-dec-2020-0000; @url:https://cov-lineages.org/global_report_B.1.351.html; @url:https://virological.org/t/tracking-the-international-spread-of-sars-cov-2-lineages-b-1-1-7-and-b-1-351-501y-v2/592]. 
B.1.351 contains several mutations at the RBD including K417N, E484K, and N501Y.

> While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640], due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].

Reviewer: ```suggestion
While the biological significance of these mutations are still under investigation, it does appear that this lineage may be associated with increased transmissibility [@doi:10.1101/2020.12.21.20248640] due to the N501Y mutation [@doi:10.1101/2020.12.14.422555; @url:https://virological.org/t/preliminary-genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-the-uk-defined-by-a-novel-set-of-spike-mutations/563].
```

>  

Reviewer: ```suggestion
```

> The P.1 variant is a branch of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travellers from Brazil during a screening at a Tokyo airport, January 10th 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].

Reviewer: ```suggestion
The P.1 variant is a branch of the B.1.1.28 lineage that was first detected in Japan in samples obtained from four travellers from Brazil during a screening at a Tokyo airport on January 10, 2021 [@url:https://www.ecdc.europa.eu/sites/default/files/documents/COVID-19-risk-related-to-spread-of-new-SARS-CoV-2-variants-EU-EEA-first-update.pdf].
```

> Shortly after it was established that there was a concentration of cases of the P.1 variant in Manaus, in the Amazonas State, Brazil.

Reviewer: ```suggestion
Shortly thereafter, it was established that there was a concentration of cases of the P.1 variant in Manaus, in the Amazonas State, Brazil.
```

> In a small subset of cases sequenced in Manaus, 42% of COVID-19 cases were identified as the P.1 variant as early as mid December, but seemed to be absent in genome surveillance testing prior to December  [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].

Reviewer: ```suggestion
In a small number of samples (n=31) sequenced in Manaus, 42% were identified as the P.1 variant as early as mid-December, but the variant seemed to be absent in genome surveillance testing prior to December [@url:https://virological.org/t/genomic-characterisation-of-an-emergent-sars-cov-2-lineage-in-manaus-preliminary-findings/586].
```

To date, at least eight countries have detected the P.1 lineage [@url:https://cov-lineages.org/global_report_P.1.html].
While the majority of P.1 cases detected internationally have been linked to travel originating from Brazil, the UK has also reported evidence of community transmission detected via routine community sequencing [@url:https://cov-lineages.org/global_report_P.1.html; @url:https://www.reuters.com/article/us-health-coronavirus-britain-variants/uk-detects-77-cases-of-south-african-covid-variant-nine-of-brazilian-idUSKBN29T07P].  

> P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD e.g. K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].

Reviewer: ```suggestion
P.1 has eight lineage-specific mutations along with three concerning spike protein mutations in the RBD, including K417T, E484K, and N501Y [@doi:10.1101/2020.12.21.20248640].
```

>  

Reviewer: ```suggestion
```

 
There have been multiple different SARS-CoV-2 lineages detected that have mostly been of no more clinical concern than the original devastating lineage originating in Wuhan [@url:https://cov-lineages.org/lineages.html].
However, the spotlight has been cast on other variants of unknown clinical relevance due to the increase of cases observed that have been associated with B.1.1.7 in particular.  
Although early in its ascendency, CAL.20C (B.1.429) is a SARS-CoV-2 variant that was detected in California, USA [@doi:10.1101/2021.01.18.21249786].
It was first detected in July 2020 but was not detected again until October 2020. 
In December 2020, CAL.20C accounted for ~24% of the total cases in Southern California and ~36% of total cases in the Los Angeles area. 

> CAL.20C has now been detected in New York, Washington DC and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].

Reviewer: ```suggestion
CAL.20C has now been detected in New York and Washington, D.C., and some cases have been reported outside the USA in Oceania [@doi:10.1101/2021.01.18.21249786].
```

This variant is characterized by five lineage-specific mutations (ORF1a: I4205V, ORF1b:D1183Y, S: S13I;W152C;L452R).

> The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012] and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].

Reviewer: ```suggestion
The latter spike mutation, L452R, is found in an area of the RBD known to resist monoclonal antibodies to the spike protein [@doi:10.1016/j.cell.2020.07.012], and it is hypothesized that this mutation may resist polyclonal sera in convalescent patients or in individuals post-vaccination [@doi:10.1101/2020.12.31.425021; @doi:10.1101/2021.01.18.21249786].
```

> CAL.20C is not a designated VOC, however, further research is required to determine the implications of the mutations encoded in this genetic variant.  

Reviewer: ```suggestion
CAL.20C is not a designated VOC; however, further research is required to determine the implications of the mutations encoded in this genetic variant.  
```

> Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany, however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any concern further [@url:https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u].

Reviewer: ```suggestion
Another notable variant has recently been discovered in 35 patients in a Bavarian hospital in Germany; however, the sequencing data has not been published to date and it remains to be determined whether this variant is of any further concern [@url:https://www.br.de/nachrichten/bayern/neue-coronavirus-mutation-35-faelle-in-garmisch-partenkirchen,SMQ1V6u].
```
